### ISSUE TITLE
Upgrade to current Huggingface version

### ISSUE THREAD

**Issue by Lukecn1** [esdurmus/feqa#7]
Hi there, 

Thank you for releasing the code for the FEQA scoring metric. 

Currently the code fails when run on the latest version of Huggingface. 

Do you have any plans on updating the repo to fix this?

**Comment by jigsaw2212** [comment]
@Lukecn1 which version of huggingface worked for you?

**Comment by Lukecn1** [comment]
> @Lukecn1 which version of huggingface worked for you? Are you able to run this repo on a GPU? I had trouble replicating the environment mentioned in the repo!

I use transformes 2.8 as specified in the environement, but it is not so straight-forward to get the environment to work. 

This is the requirements I used to get it to work: 

-f https://download.pytorch.org/whl/cu101/torch_stable.html
torch==1.5.0 
torchvision==0.6.0
Cython==0.29.15
numpy==1.19.1
fairseq==0.10.0
nltk==3.5
spacy==2.3.2
tensorflow==1.15.2
transformers==2.8.0
benepar==0.1.2
